doc/Quick_start/upgrading.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading existing deployment
```

Comment:
I'd add a note that all exisitng network and network configuration will be wiped at the beginning rather than in futher instuctions.